test/fakeEthInfo.ts

```diff
@@ -4,7 +4,7 @@ import { FakeSettings } from './fakeCurrencyPlugin'
 
 const defaultSettings: FakeSettings = {
   customFeeSettings: ['gasLimit', 'gasPrice'],
-  publicAddress: '0x0d73358506663d484945ba85d0cd435ad610b0a0',
+  publicAddress: '0x6f8bc2d82175489f50537d868bcf2e4f15c3fa95',
```

Comment:
True, but also not very relevant. These are only used for the testpartner.ts utility development script. This was changed to match real public addresses that have balances in them in case the Lifi API was checking. I'll revert back.